_posts/blog/2016-12-19-running-on-fedora-25.md

```diff
@@ -0,0 +1,335 @@
+---
+title: "Running Accumulo on Fedora 25"
+author: Christopher Tubbs and Mike Miller
+reviewers: 
+---
+
+Apache Accumulo has been available in [Fedora] for the last few releases.
+Recently, the [maintainers] have updated it to version `1.6.6` and have made
+some improvements to the packaging and launch scripts to provide a good
+out-of-box experience. This post will discuss the basic setup procedures for
+running Accumulo in the latest version, `Fedora 25`.
+
+## Prepare the system
+
+**WARNING**: Before you start, be sure you've got plenty of free disk space.
+Otherwise, you could run into this [bug] or see other problems.
+
+These instructions will assume you're using Fedora 25, fully up-to-date (`sudo
+dnf --refresh upgrade`).
+
+### Install packages
+
+The Fedora maintainers provide a meta-package to install accumulo and all of
+its dependencies. We also recommend installing the JDK, so you'll have access
+to the `jps` command, and `tuned` for setting system performance tuning
+parameters from a profile. We'll also make sure our preferred editor is
+installed (replace `vim` with your preferred editor):
+
+```bash
+sudo dnf install accumulo java-1.8.0-openjdk-devel tuned vim
+```
+
+It is possible to install only a specific Accumulo service, if desired, but it
+won't hurt to have extra packages installed and not used. For the single node
```

Comment:
could drop `if desired, but it won't hurt to have extra packages installed and not used`